skymap_scanner/server/start_scan.py

```diff
@@ -448,7 +448,7 @@ def _gen_pixel_variations(
 
         for i in range(0,len(self.pos_variations)):
             posVariation = self.pos_variations[i]
-            if self.reco_algo != 'millipede':
+            if self.reco_algo != 'millipede_original':
```

Comment:
My intuition is this could cause a future pain point. Can you make this explicit, i.e. use `==` only?

Reply:
Another option id to move this logic to the `RecoInterface`-based classes

Reply:
Good catch. I changed it to `==` for now.